discovery-provider/docker-compose.yml

```diff
@@ -262,6 +262,8 @@ services:
     restart: unless-stopped
     env_file:
       - ${OVERRIDE_PATH:-override.env}
+    environment:
+      METABASE_PORT: ${METABASE_PORT:-3000}
```

Comment:
Changing this port when a previous metabase instance was running is kind of finicky, i've had to restart postgres alongside metabase to get this to work and would recommend setting this once and not changing it again.

Reply:
Wait I actually don't think this port needs to be configurable. I was referring to the concept of pointing the root server to backend:5000 or metabase:3000